lib/normalize_country/countries/en.yml

```diff
@@ -461,7 +461,11 @@ CD:
   - Congo-Kinshasa
   - DRC
   - DR Congo
-  - Congo, The Democratic Republic Of The
+  - Congo, Democratic Republic of
```

Comment:
We should keep `Congo, The Democratic Republic Of The`. It's valid: https://www.iso.org/obp/ui/#iso:code:3166:CD. 


Version with parenthesis should be added too but this looks like a broader task for another day.

Reply:
I think I meant to add the extra alias, and not delete the existing one. Will fix.